configs/gatk_sv/config_overrides.toml

```diff
@@ -8,3 +8,9 @@
 
 [resource_overrides.GatherSampleEvidence.runtime_attr_scramble_part2]
 mem_gb = 7.5
+
+[resource_overrides.EvidenceQC.wgd_build_runtime_attr]
```

Comment:
@vivbak, just flagging resource overrides will potentially be a challenge if we move to structured configs.